skyportal/handlers/api/group.py

```diff
@@ -468,12 +468,12 @@ def post(self, group_id, *ignored_args):
             return self.error("Specified stream_id does not exist.")
         else:
             # todo: ensure all current group users have access to this stream
-            for user in group.users:
-                # get single user group
-                single_user_group = Group.query.filter(  # noqa
-                    Group.name == user.username
-                ).first()
-                # todo: do the check
+            # for user in group.users:
```

Comment:
Delete this block?

Reply:
@dmitryduev okay to delete this commented-out block? I believe it was part of your PR

Reply:
Done